### DISCUSSION TITLE
Tokenize Title / description

### DISCUSSION THREAD

**Discussion by Tychodewaard** [vanjarosoftware/Vanjaro.Platform#1730] [Q&A]
If you want to have all detail pages within an application to have a different title/description, DNN has a problem. MyTokens solves this as on https://docs.dnnsharp.com/my-tokens/how-to/tokenize-title-meta.html

But how to handle this in Vanjaro?
Because I don't think Vanjaro is built to register extra stuff :-)

**Comment by meetmandeep** [comment]
> If you want to have all detail pages within an application to have a different title/description, DNN has a problem. 

It's super easy to change the title/description of detail pages within an application from server side code. App developer should be able to accomplish this easily. There are built in properties that can be overwritten such as:

 (DotNetNuke.Framework.CDefault)this.Page.Title = "Your custom page title here";

Or you can always find the the Meta and Title tags in Page object and manipulate.
